frontend/src/metabase-lib/v1/expressions/config.ts

```diff
@@ -303,6 +291,18 @@ export const MBQL_CLAUSES: MBQLClauseMap = {
     args: ["expression", "expression"], // ideally we'd alternate boolean/expression
     multiple: true,
   },
+  offset: {
+    displayName: `Offset`,
+    type: "any", // ideally we'd dynamically infer it from the first argument
```

Comment:
I moved the definition from aggregations section closer to similar functions (`coalesce` and `case`) and changed the return `type` to `"any"`, since `offset` will return the same type as it receives as the first argument (which is `"any"`).